ros2/nightly/nightly/prereqs.yaml

```diff
@@ -2,8 +2,5 @@ console_bridge: {ubuntu: []}
 fastcdr: {ubuntu: []}
 fastrtps: {ubuntu: []}
 osrf_testing_tools_cpp: {ubuntu: []}
-poco_vendor: {ubuntu: []}
-tinyxml_vendor: {ubuntu: []}
-tinyxml2_vendor: {ubuntu: []}
-urdfdom: {ubuntu: []}
+tinydir_vendor: {ubuntu: []}
```

Comment:
Do we still need this if we are installing `ros-$ROS_DISTRO-ros-workspace`?

Reply:
yes. It is the other way around, we can afford to add this line now that we install `ros-$ROS_DISTRO-ros-workspace`.

---

Once #245 merged, there should be no `ros-crystal-*` packages installed in the nightly image except from `tinydir_vendor` and `ros_workspace`.

With the nightly PR installing `ros-$ROS_DISTRO-ros-workspace`, we do not need to install any other ros-crystal package from debs. So we can stop installing that one remaining package.

As of why this package needs to be listed, my guess it that it's a pure cmake package and not an ament package, so it doesnt install it's package.xml nor register itself in the ament index.

Reply:
@cottsay contributed https://github.com/ros2/tinyxml2_vendor/pull/6 which resolves this issue for tinyxml2_vendor. Something similar could be done for tinydir_vendor.

Reply:
:+1: that makes sense for vendor packages as it doesn't really matter if they're not registered in the ament index.